tasks/tests/test_base.py

```diff
@@ -0,0 +1,29 @@
+# -*- coding: utf-8 -*-
+from tasks.base import check_response
+from requests import Response
```

Comment:
Same comment as above, it would be best to switch the order of these imports and put a blank line in between. Also, I like to put a docstring at the top of the file to briefly explain what it's for, like:

```
# -*- coding: utf-8 -*-
""" This file tests the classes and functions in tasks/base.py """
# imports after that
``` 

This is a "Sandy standard", not a community standard, so you can feel free to ignore if you like! 😄